src/web/layouts/CommentLayout.tsx

```diff
@@ -216,12 +216,10 @@ export const CommentLayout = ({
 
     const adTargeting: AdTargeting = buildAdTargeting(CAPI.config);
 
-    // Render the slot if one is true:
-    // 1) The flag for this slot exists in the URL (i.e. ?slot-machine-flags=showBodyEnd)
-    // 2) The global switch for the Frontend/DCR Epic test is true
+    // Render the slot if switch is set to true or flag set in URL (i.e. ?slot-machine-flags=showBodyEnd)
```

Comment:
This is a case where the code is clear enough that a comment doesn't add much.

Reply:
Fair enough, I've removed those